sound/audioclass.py

```diff
@@ -95,34 +55,27 @@ def GetAudioClass(self, data):
         else:
             d.popleft();d.append(self.Decibel)
             self.loop += 1
+        change = data.suddenchange
+        print "sudden sound change value {}".format(data.suddenchange)
+    	
+        loud = '(StateLink Sudden sound change value (NumberNode "' + \
```

Comment:
If I use this:
loud = '(StateLink Sudden sound change value (NumberNode "' + \
		       str(change) + '"))'
and put this one in self-model.scm:
(define loud-sound  (AnchorNode "Sudden sound change value"))
wouldn't that be the same as putting it as AnchorNode, or do I have to make it define-public?